<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qimg2lcd</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="text">
         <string>Bild Laden</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExport">
        <property name="text">
         <string>LCD Exportieren</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="lblRed">
          <property name="text">
           <string>Rot</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblGreen">
          <property name="text">
           <string>Grün</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblBlue">
          <property name="text">
           <string>Blau</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="slidRed">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>128</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="slidGreen">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>128</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="slidBlue">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>128</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="lblHight">
          <property name="text">
           <string>Höhe</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinHight">
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="value">
           <number>68</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblWidth">
          <property name="text">
           <string>Breite</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinWidth">
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="value">
           <number>96</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblImage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>2000</width>
          <height>2000</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>20000</width>
          <height>20000</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Bild</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblPreview">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>2000</width>
          <height>2000</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>20000</width>
          <height>20000</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Vorschau</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnLoad</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OpenFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnExport</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ExportFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinHight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>ChangeSize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinWidth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>ChangeSize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slidRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>UpdatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slidGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>UpdatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slidBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>UpdatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OpenFile()</slot>
  <slot>ExportFile()</slot>
  <slot>ChangeSize()</slot>
  <slot>UpdatePreview()</slot>
 </slots>
</ui>
